nfcttrace.bt

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bpftrace
+/*
+ * nfcttrace.bt     Show entries about TCP&UDP in nf_conntrack. For Linux, uses bpftrace, eBPF.
+ *
+ * 7-Jul-2023   Weiyang Wang	Created this.
```

Comment:
We're also not going to maintain a pseudo-changelog in the script headers in this repository as it's not a reliable source of information (some people update it, some don't) and to avoid debates on what changes are worth including. The Git logs should provide a complete history.

The copyright notice will of course remain.

```suggestion
```